packages/template-retail-react-app/app/utils/product-utils.js

```diff
@@ -51,40 +51,15 @@ export const getPriceData = (product, opts = {}) => {
     let variantWithLowestPrice
     // grab the variant that has the lowest price (including promotional price)
     if (isMaster) {
-        const variants = product?.variants || []
-        variantWithLowestPrice = variants.reduce(
-            (minVariant, variant) => {
-                const promotions = variant.productPromotions || []
-                const smallestPromotionalPrice = getSmallestValByProperty(
-                    promotions,
-                    'promotionalPrice'
-                )
-                const variantSalePrice =
-                    smallestPromotionalPrice && smallestPromotionalPrice < variant.price
-                        ? smallestPromotionalPrice
-                        : variant.price
-                return variantSalePrice < minVariant.minPrice
-                    ? {minPrice: variantSalePrice, variant}
-                    : minVariant
-            },
-            {minPrice: Infinity, variant: null}
-        )
+        variantWithLowestPrice = findLowestPrice(product)
         currentPrice = variantWithLowestPrice?.minPrice
     } else {
-        const promotionalPrice = getSmallestValByProperty(
-            product?.productPromotions,
-            'promotionalPrice'
-        )
-        currentPrice =
-            promotionalPrice && promotionalPrice < product?.price
-                ? promotionalPrice
-                : product?.price
+        currentPrice = findLowestPrice(product)?.minPrice
     }
 
     // since the price is the lowest value among price books, each product will have at lease a single item tiered price at quantity 1
     // the highest value of tieredPrices is presumptively the list price
-    const tieredPrices =
-        variantWithLowestPrice?.variant?.tieredPrices || product?.tieredPrices || []
+    const tieredPrices = variantWithLowestPrice?.data?.tieredPrices || product?.tieredPrices || []
```

Comment:
@alexvuong it's because I also updated it:
from `variantWithLowestPrice?.variant` to `variantWithLowestPrice?.data`

The line becomes shorter as a result and can now fit into 1 line.